src/main/java/sirius/biz/storage/layer1/ObjectStorageSpace.java

```diff
@@ -481,6 +482,28 @@ public void deliver(Response response, String objectId, boolean largeFileExpecte
         }
     }
 
+    /**
+     * Determines if a special delivery for very large files should be attempted.
+     * <p>
+     * For very large files (multiple GBs), we use special URLs generated by the
+     * {@link sirius.biz.storage.layer2.URLBuilder} and handled by the {@link sirius.biz.storage.layer2.BlobDispatcher}.
+     * These contain the "/xxl/" marker so that downstream caches like <tt>Varnish</tt> don't even attempt to put the
+     * data in their cache. These files therefore are piped through which may lead to increasingly large internal
+     * buffers as an upstream storage like <b>S3</b> might send data faster than we pump through to slower clients
+     * like mobile users. We therefore use a blocking/stream based approach in a thread pool of limited size. However,
+     * in the presence of a <tt>Range</tt> header, the date to transfer is most probably way smaller and also the stream
+     * implementation cannot satisfy such requests. We therefore only use this approach is a large file is expected and
+     * no <tt>Range</tt> header is present.
+     *
+     * @param response          the response to handle
+     * @param largeFileExpected a flag which determines if a large file is expected at all
+     * @return <tt>true</tt> if the delivery should be handled in a separate thread pool and a non-blocking fashion,
+     * <tt>false</tt> otherwise
```

Comment:
```suggestion
     * @return <tt>true</tt> if the delivery should be handled in a separate thread pool and a non-blocking fashion
```
In my opinion this its redundant to tell that a boolean parameter can also be false :)